engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/SelectGameScreen.java

```diff
@@ -83,6 +84,11 @@ public String get() {
 
         refreshList(gameList);
         gameList.subscribe((widget, item) -> loadGame(item));
+        gameList.subscribeSelection((widget, item) -> {
+            for (String targetId : new String[]{"load", "delete"}) {
+                find(targetId, UIButton.class).setEnabled(true);
```

Comment:
Does that still work if you close the menu and open it again? I would probably tie the selection explicitly to the question whether an item in the list is selected or not. This will take care of disabling them at the right time, too.

Reply:
`.setEnable(item != null)` is probably all change needed here.